view/base.js

```diff
@@ -352,9 +355,9 @@ if ( typeof kintInitialized === 'undefined' ) {
 
 			if ( keyCode === 9 ) { // TAB : moves up/down depending on shift key
 				return kint.keyCallBacks.moveCursor(shiftKey, i);
-			} else if ( keyCode === 38 ) { // ARROW UP : moves up
+			} else if ( keyCode === 38 || keyCode === 75 ) { // ARROW UP : moves up
```

Comment:
This as well as other places should reflect your change in the comments, cause keyCode 75 is highly unreadable. Please observe the line limit as well.